### ISSUE TITLE
OpenShift-Route Parameter "caCertificate" is ignored by CIS

### ISSUE THREAD

**Issue by joebride** [F5Networks/k8s-bigip-ctlr#1631]
**Setup Details**

CIS Version : 2.2.1
Build: f5networks/k8s-bigip-ctlr:latest
BIGIP Version: Big IP 15.1.0.3
AS3 Version: 3.24.0
Agent Mode: AS3
Orchestration: K8S/OSCP v1.17.1, OS v4.2
Orchestration Version:
Pool Mode: Cluster
Additional Setup details: OpenShift-Routes

**Description**
When I add OS-Route parameter "caCertificate" the CIS ignore this modification and do not send proper AS3-declaration to BIG-IP.

**Expected Result**
When specific OS-Route requires a certificate chain (specified by parameter "caCertificate") the CIS must add parameter (TLS_server - "chainCA":) to the AS3-declaration sent to BIG-IP.
See also:  https://github.com/F5Networks/f5-appsvcs-extension/issues/285

**Actual Result**
CIS ignore this parameter "caCertificate"

**Excerpt CIS-config**
```
image: 'registry.connect.redhat.com/f5networks/cntr-ingress-svcs:2.2.1-rhel7'
args:
- '--bigip-username=$(BIGIP_USERNAME)'
- '--bigip-password=$(BIGIP_PASSWORD)'
- '--bigip-url=A.B.C.D'
- '--bigip-partition=XXX'
- '--pool-member-type=cluster'
- '--openshift-sdn-name=/Common/RD_000_TUN_VXL_0001'
- '--log-level=INFO'
- '--log-as3-response=true'
- '--manage-routes=true'
- '--manage-ingress=false'
- '--route-label=XXX'
- '--route-http-vserver=RD_000_VS_RED_XXX_80'
- '--route-https-vserver=RD_000_VS_STD_XXX_443'
- '--route-vserver-addr=A.B.C.D'
- '--agent=as3'
- '--insecure=true'
- '--tls-version=1.3'
- '--cipher-group=/Common/CIPHER_GROUP_AES-GCM_ECDHE_DHE_TLSV12_TLSV13'

```

**Comment by trinaths** [comment]
@joebride - Can you share Route examples where you want to use a CaCertificate. 

One such use can be, 
`apiVersion: v1
kind: Route
metadata:
  name: frontend
spec:
  host: www.example.com
  to:
    kind: Service
    name: frontend
  tls:
    termination: edge
    key: |-
      -----BEGIN PRIVATE KEY-----
      [...]
      -----END PRIVATE KEY-----
    certificate: |-
      -----BEGIN CERTIFICATE-----
      [...]
      -----END CERTIFICATE-----
    caCertificate: |-
      -----BEGIN CERTIFICATE-----
      [...]
      -----END CERTIFICATE-----`

**Comment by joebride** [comment]
@trinaths - Yes, this is the proper use-case.

**Comment by trinaths** [comment]
Thanks @joebride. Created JIRA CONTCNTR-2396 for PM tracking.

**Comment by trinaths** [comment]
This issue is fixed with CIS v2.3
